src/ai/backend/storage/config/unified.py

```diff
@@ -609,6 +609,64 @@ class PresignedDownloadConfig(BaseModel):
     )
 
 
+class VFSStorageConfig(BaseModel):
+    name: str = Field(
+        description="""
+        Name of the VFS storage configuration.
+        Used to identify this storage in the system.
+        """,
+        examples=["local-models", "shared-datasets"],
+    )
+    type: Literal["vfs"] = Field(
+        description="""
+        Storage type identifier for VFS storage.
+        Must be 'vfs' for VFS storage configurations.
+        """,
+        default="vfs",
+    )
```

Comment:
Is `type` needed in `vfsStorage`?

Reply:
It's useless in the current config structure.
We need to decide on the storage config structure first